library/Icingadb/Util/PerfData.php

```diff
@@ -312,7 +320,7 @@ public function isCounter(): bool
      */
     public function isVisualizable(): bool
     {
-        return isset($this->minValue) && isset($this->maxValue) && isset($this->value);
+        return isset($this->minValue) && isset($this->maxValue) && isset($this->value) && ! $this->isInvalid();
```

Comment:
```suggestion
return isset($this->minValue, $this->maxValue, $this->value) && ! $this->isInvalid()
```

Reply:
Well this sure is practical. I just wanted to  add the condition`! $this->isInvalid()`. But I can make the changes.